src/components/Send/CollaboratorsSelector.tsx

```diff
@@ -78,7 +81,7 @@ const CollaboratorsSelector = ({
           max={maxNumCollaborators}
           isInvalid={usesCustomNumCollaborators && !isValidNumCollaborators(field.value, minNumCollaborators)}
           placeholder={t('send.input_num_collaborators_placeholder')}
-          value={customNumCollaboratorsInput || ''}
+          value={customNumCollaboratorsInput}
```

Comment:
```suggestion
          value={customNumCollaboratorsInput || ''}
```

Reply:
I removed it because customNumCollaboratorsInput will always be initialised:
```
const initialNumCollaboratorsInput = isDevMode() ? '1' : ''

const [customNumCollaboratorsInput, setCustomNumCollaboratorsInput] = useState<string>(initialNumCollaboratorsInput)
```
Hence, the fallback will be obsolete, right? Or should I add it back for some other reason / edge case?

Reply:
> Also, can we use `undefined` (not `''` empty string) for `customNumCollaboratorsInput` when a user did not touch the input field yet?

if we make `customNumCollaboratorsInput` to  `undefined` then we'll need the fall back you're right so ill add it again!